src/main/java/com/kodilla/ecommercee/domain/product/Product.java

```diff
@@ -0,0 +1,28 @@
+package com.kodilla.ecommercee.domain.product;
+
+public class Product {
```

Comment:
Do czego ta klasa ma służyć? Nie jest używana przez kontroler. Jeśli chodzi o implementacje Product(jako entity) to jest na to inny task

Reply:
poprawione